packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -258,6 +260,45 @@ export default function makeKernelKeeper(
     kvStore.set('crankNumber', `${crankNumber + 1n}`);
   }
 
+  function initQueue(queue) {
+    kvStore.set(`${queue}`, JSON.stringify([1, 1]));
```

Comment:
part of me wants for these to be BigInts.. `2^53` cranks at (an aspirational) 1000/sec gives us about 300kyears, which is obviously plenty but also not as close to infinity as I'd like

we'd need something other than `JSON.stringify`, like `s.split(',').map(BigInt)` and a `${head},${tail}`